src/conditionalAndLoops.js

```diff
@@ -19,8 +19,20 @@
  *
  */
 function getFizzBuzz(num) {
-	throw new Error("Not implemented");
+  if (num % 3 == 0 && num % 5 == 0) {
+    return "FizzBuzz"
+  }
+  else if (num % 3 == 0) {
+    return "Fizz"
+  }
+  else if (num % 5 ==0) {
+    return "Buzz"
+  }
+  else {
+    return num
+  }	
 }
+getFizzBuzz(15);
```

Comment:
Remove or comment out these calls to functions. They interfere with the testing script.